batch_uploads_imageuploader

```diff
@@ -1,4 +1,49 @@
 #!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+batch_uploads_imageuploader -- a script that runs imaging_upload_file.pl in
+batch mode
+
+=head1 SYNOPSIS
+
+./batch_uploads_imageuploader -profile prod < list_of_scans.txt >log_batch_imageuploader.txt 2>&1 `[options]`
```

Comment:
Should probably add the .pl to the script, as for the tarchiveLoader.

Reply:
as decided, we will do this separately; RM#13755